rails/Gemfile

```diff
@@ -0,0 +1,10 @@
+source 'https://rubygems.org'
+
+gem 'benchmark-ips', github: 'kirs/benchmark-ips', branch: 'quiet-mode'
```

Comment:
We need IPS to [support the `quiet` option](https://github.com/evanphx/benchmark-ips/compare/master...kirs:quiet-mode). I will send a PR to the upstream soom.